lib/Optimizer/Transforms/DecompositionPatterns.cpp

```diff
@@ -1181,6 +1181,41 @@ struct RxToPhasedRx : public OpRewritePattern<quake::RxOp> {
   }
 };
 
+// quake.rx<adj> (θ) target
+// ─────────────────────────────────
+// quake.rx(-θ) target
```

Comment:
These adjoint forms are a bit surprising. Do you know what is generating them? The code that generates generalized adjoints doesn't use these. It just negates the angle argument.

I think we should probably get rid of these `r?<adj>` as a valid operation. We did that with `x<adj>` and such.

Reply:
I also found that some decomposition patterns create them, "CRxToCX" for example

Reply:
chatted offline, will postpone the `adjoint` removal for later